x-pack/plugins/fleet/common/types/models/uninstall_token.ts

```diff
@@ -8,5 +8,7 @@
 export interface UninstallToken {
   policy_id: string;
   token: string;
-  created_at?: string;
+  created_at: string;
 }
+
+export type UninstallTokenMetadata = Omit<UninstallToken, 'token'>;
```

Comment:
I wanted to emphasise that there is no real token here, only some description about it, like creation date. therefore I think `UninstallTokensList` would be a bit misguiding: I'd assume that I receive a list of tokens, not only their descriptions. so I'd leave it as metadata, or something else that indicate that there is no token. what do you think?

Reply:
Yeah, agreed that `UninstallTokenList` might be confusing. I would avoid `metadata` only because we are not populating the response with extra data. Maybe something related to the policyIds history?

Reply:
I wasn't sure, but based on definitions I found, I believe `metadata` indicates the intention perfectly here - it's about describing our data, the token, with information like creation date and id. I have no better idea really - `UninstallTokenDescription`, `LatestUninstallTokenData`, these come to my mind, but they don't help I think. What do you say about keeping it this way?